### DISCUSSION TITLE
RN 0.70 - Error

### DISCUSSION THREAD

**Discussion by megengels** [sindresorhus/ky#478] [Q&A] [answered]
Hi everybody,

 I'm working on a mobile application with RN and I need to use an API.

 As a lot of people, i have implement Axios but i want to try to use KY.

 Unfortunately, i have a error message and I found only one issues of this subject :

#466 -> 
_Unfortunately, this PR is broken on react native (tested on RN 0.69.6 with hermes engine).
Error: FileReader.readAsArrayBuffer is not implemented is being thrown.
A bit of pity, because until this PR it's been working flawlessly on RN._

 There is somebody to help me please ?

Thank's a lot !

**Comment by davidrunger** [comment, marked as answer]
There is an open issue about this at the react-native repo here: https://github.com/facebook/react-native/issues/34402. It seems to be a bug / missing feature of react-native. The ultimately best way forward will probably be for that react-native issue to be resolved, which should then fix the incompatibility that now exists between react-native and the latest versions of ky. Short of that, I'm not sure what the best way to resolve this might be, or whether it's worthwhile to attempt to resolve.

**Reply by megengels** [reply]
Hi,
 Thank you for that answer. Indeed, I did not think to look for the RN problem.
 I hope the RN team will do the ncessary soon as I would really like to use KY instead of Axios..
 Pending, I should continue using Axios.

see you soon ;-)
